docs/source/specs/openapi.json

```diff
@@ -1767,7 +1767,7 @@
           {
             "name": "application",
             "in": "query",
-            "description": "Partial match for the application name of a permission.",
+            "description": "Partial match for the application name of a permission. You may also use a comma-separated list to match on multiple applications.",
```

Comment:
Does the partial match description still valid after this change? If there is a unit test to ensure this would be great!

Reply:
Ah, good catch! No, the partial match would no longer work, here. I don't believe that was actually a requirement of the `/permissions/` endpoint, but I'm going to confirm.

Reply:
Confirmed from Robin that this is not an expectation/requirement. It was just a side-effect of implementation, so I'm going to remove the documentation for this support. Thanks, @astrozzc!

Reply:
👍great！